mscore/inspector/inspectorMeasureNumber.cpp

```diff
@@ -11,8 +11,7 @@
 //=============================================================================
 
 #include "inspectorMeasureNumber.h"
-#include "libmscore/score.h"
-#include "icons.h"
+#include "libmscore/measureNumber.h"
```

Comment:
Should be `#include "libmscore/measurenumber.h"`, `n` lower case.

Reply:
hum. Weird it builds on my machine...

Reply:
it also built on appveyor... but not on travis... I am not sure I like all this black magic...